published/012-2018-Oct-2.md

```diff
@@ -0,0 +1,63 @@
+## Welcome to the IPFS Weekly. 👋
+
+The [InterPlanetary File System (IPFS)](https://ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identity. IPFS enables the creation of completely distributed applications. It aims to make the web faster, safer, and more open. Since that’s a pretty large scope, we track development across the ecosystem in this weekly dispatch.
+
+Looking to get involved? Click on some of the links below, see what we’re up to on [GitHub](https://github.com/ipfs), or join us on [IRC](https://webchat.freenode.net/?channels=ipfs_) (#ipfs on the Freenode network).
```

Comment:
The irccloud channel invite page is marginally more friendly
https://www.irccloud.com/invite?channel=%23ipfs&hostname=irc.freenode.net&port=6697&ssl=1

and it offers a "join with other IRC client" link so it's not too bad on the sass silo front either.

Maybe its some magic of how the webchat link works, but it looks like it links to `#ipfs_` with at trailing underscore rather than `#ipfs`

Reply:
nerp, i just tried it out and it takes me too a private nether realm all on my own!

Reply:
We can also just link to Matrix/Riot, #ipfs is bridged: https://riot.im/app/#/room/#ipfs:matrix.org